packages/config/src/chainConfig/networks/gnosis.ts

```diff
@@ -33,6 +33,8 @@ export const gnosisChainConfig: ChainConfig = {
   // Networking
   MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS: 16384,
 
+  BLOB_SCHEDULE: [{EPOCH: 1337856, MAX_BLOBS_PER_BLOCK: 2}],
```

Comment:
electra is missing, I will push an update for this

also noticed gnosis config is inconsistent when it comes to deneb, see [here](https://github.com/gnosischain/configs/blob/d447a94bbe1bafbef4fe5a2fbb2e5469e7b494bb/mainnet/config.yaml#L139-L142)

`MAX_REQUEST_BLOB_SIDECARS` value is wrong